main.py

```diff
@@ -1,10 +1,58 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def get_race(race_data: dict) -> Race:
+    race, _ = Race.objects.get_or_create(
+        name=race_data.get("name"),
+        description=race_data.get("description")
+    )
+    return race
+
+
+def create_skills(skills_data: dict, race: Race) -> None:
+    for skill in skills_data:
+        if not Skill.objects.filter(name=skill.get("name")).exists():
+            Skill.objects.create(
```

Comment:
You can use get_or_create
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create
Fix in all places